src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
```

Comment:
There is no need to add "ball**" prefix for fields because they are already fields of the "Ball" class, and it is already clear that they are Ball's fields.